pinot-distribution/pinot-assembly.xml

```diff
@@ -83,6 +83,10 @@
       <source>${pinot.root}/pinot-plugins/pinot-file-system/pinot-hdfs/target/pinot-hdfs-${project.version}-shaded.jar</source>
       <destName>plugins/pinot-file-system/pinot-hdfs/pinot-hdfs-${project.version}-shaded.jar</destName>
     </file>
+    <file>
+      <source>${pinot.root}/pinot-plugins/pinot-file-system/pinot-s3/target/pinot-s3-${project.version}-shaded.jar</source>
```

Comment:
Would be good to update the commit log stating `pinot-s3` as well.

Reply:
I just re-order pinot-s3 to the right section.